draft-kaippallimalil-tsvwg-media-hdr-wireless.md

```diff
@@ -20,7 +20,7 @@ venue:
   type: Working Group
   mail: tsvwg@ietf.org
   arch: https://mailarchive.ietf.org/arch/browse/tsvwg/
-  github: https://github.com/SpencerDawkins/media-hdr-wireless
+  github: SpencerDawkins/media-hdr-wireless
```

Comment:
**WOW**. @danwing, thank you for spotting that! I didn't realize what @martinthomson/i-d-template was prefixing the URL I was providing (so the result was **https://github.com/https://github.com**/SpencerDawkins/media-hdr-wireless). It's fine that it works that way, it's just that arch: doesn't work the same way. I'm merging this, and going off to see what my other drafts look like ....